NewEssentials/Chat/BroadcastingService.cs

```diff
@@ -0,0 +1,73 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Cysharp.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using OpenMod.API.Ioc;
+using OpenMod.API.Plugins;
+using OpenMod.Core.Helpers;
+using SDG.Unturned;
+using UnityEngine;
+
+namespace NewEssentials.Chat
+{
+    [ServiceImplementation]
+    public class BroadcastingService : IBroadcastingService
```

Comment:
At some point I would like to inherit IAsyncDisposable here and clear effects on disposal. You can implement it yourself or add a TODO